features/copyCourse.feature

```diff
@@ -1,108 +1,80 @@
-        And teacher chooses course with name '<courseName>'
-        And teacher adds a topic with name '<topicName>'
-        And teacher adds content Text with title '<contentTitle>' and description '<contentDescription>'
-        And teacher clicks Save-changes button
-        And teacher goes to courses page
-        And teacher chooses course with name '<courseName>'
-        And teacher clicks Duplicate-course button
-        And teacher goes to courses page
-        Then teacher should see that course with name '<courseName> - Kopie' is visible on the list
-        And teacher should see that copied course with name '<courseName> - Kopie' contains topic with name '<topicName>'
-        Examples:
-            | courseName    | topicName         | contentTitle      | contentDescription |
-            | sample course | sample topic name | test text content | test description   |
-
-
-    @copyCourseWithGeoGebra
-    Scenario Outline: teacher can copy course with certain GeoGebra
-        Given teacher creates course with name '<courseName>'
-        And teacher chooses course with name '<courseName>'
-        When teacher adds a topic with name '<topicName>'
-        And teacher adds content GeoGebraArbeitsblatt with id '<geogebraID>'
-        And teacher clicks Save-changes button
-        And teacher goes to courses page
-        And teacher chooses course with name '<courseName>'
-        And teacher clicks Duplicate-course button
-        And teacher goes to courses page
-        Then teacher should see that cloned course with name '<courseName> - Kopie' is visible on the list
-        And teacher should see that copied course with name '<courseName> - Kopie' contains topic with name '<topicName>'
-        Examples:
-            | courseName        | topicName | text                  | geogebraID |
-            | sample course geo | geo topic | some sample text here | ucxngdjf   |
+Feature: Copy a created course with different data
 
-    #@copyCourseWithMaterial
-    #Scenario Outline: teacher can copy course with certain Material
-    #Given teacher creates course with name '<courseName>'
-    #And teacher chooses course with name '<courseName>'
-    #And teacher adds a topic with name '<topicName>'
-    #When teacher adds content Material
-    #And teacher clicks Save-changes button
-    #And teacher goes to courses page
-    #And teacher chooses course with name '<courseName>'
-    #And teacher clicks Duplicate-course button
-    #And teacher goes to courses page
-    #Then teacher should see that copied course with name '<courseName> - Kopie' is visible on the list
-    #And teacher should see that copied course with name '<courseName> - Kopie' contains topic with name '<topicName>'
-    #Examples:
-    #| courseName             | topicName      | text                  |
-    #| sample course material | material topic | some sample text here |
+	Background:
+		Given User arrives on the Schul-Cloud homepage
```

Comment:
Shouldn't the user be parameterized here too?

Reply:
In background area we can not used parametrization.

Reply:
I agree with Iza because It was agreed that we write the parameters inside ''.

Reply:
okay, good to know